src/metabase/public_settings.clj

```diff
@@ -71,6 +71,17 @@
   :type    :boolean
   :default true)
 
+(defsetting reset-token-ttl-h
```

Comment:
nit: I'd prefer writing out the unit as `reset-token-ttl-hours`. Wasn't obvious to me what it stood for at first (unlike `ms` which is more self-evident)

Reply:
Could you also move this to `metabase.api.session` near where it's being used? If it's not a public setting it doesn't need to be in this namespace. (I know, we're not consistent about that, but we don't need to make the problem worse)

Reply:
done